pinot-clients/pinot-java-client/src/main/java/org/apache/pinot/client/ResultSetGroup.java

```diff
@@ -28,6 +28,7 @@
  */
 public class ResultSetGroup {
   private final List<ResultSet> _resultSets;
+  private final ResultSetStats _resultSetStats;
```

Comment:
This should be a List as well since ResultSet is a list

Reply:
A single query only has one set of stats. AFAIK, `ResultSet` is a list because Pinot sends a separate set of results for each group when there is `groupBy` but that doesn't mean there will be multiple response stats.

@fx19880617 correct me if my understanding isn't correct.

Reply:
@buchireddy that's right. we don't need it to be part of each resultset

Reply:
cool.